diff --git a/polyclipping.spec b/polyclipping.spec
index 760bcad..edc8d49 100644
--- a/polyclipping.spec
+++ b/polyclipping.spec
@@ -6,13 +6,14 @@
 
 Name:           polyclipping
 Version:        5.1.6
-Release:        1%{?dist}
+Release:        2%{?dist}
 Summary:        Polygon clipping library
 
 Group:          System Environment/Libraries
 License:        Boost
 URL:            http://sourceforge.net/projects/polyclipping
 Source0:        http://downloads.sourceforge.net/%{name}/clipper_ver%{version}.zip
+Patch0:         %{name}-bug47.patch
 
 BuildRequires:  cmake
 BuildRequires:  dos2unix
@@ -38,6 +39,7 @@ developing applications that use %{name}.
 
 %prep
 %setup -qc
+%patch0 -p1
 
 # Delete binaries
 find . \( -name "*.exe" -o -name "*.dll" \) -delete
@@ -84,6 +86,9 @@ popd
 %{_libdir}/lib%{name}.so
 
 %changelog
+* Tue Jun 25 2013 Miro Hrončok <mhroncok@redhat.com> - 5.1.6-2
+- Added patch to solve upstream bug 47
+
 * Wed Jun  5 2013 Tom Hughes <tom@compton.nu> - 5.1.6-1
 - Update to 5.1.6 upstream release
 - Install agg_conv_clipper.h
