diff -ru a/libraries/Makefile.am b/libraries/Makefile.am
--- a/libraries/Makefile.am	2012-10-16 05:06:27.000000000 +0200
+++ b/libraries/Makefile.am	2013-01-31 20:54:29.859650259 +0100
@@ -1,6 +1,4 @@
 include libraries/lmfit/Makefile.am
 include libraries/libreprap/Makefile.am
-include libraries/vmmlib/Makefile.am
-include libraries/clipper/Makefile.am
 include libraries/poly2tri/Makefile.am
 include libraries/amf/Makefile.am
diff -ru a/src/Makefile.am b/src/Makefile.am
--- a/src/Makefile.am	2012-10-16 05:06:27.000000000 +0200
+++ b/src/Makefile.am	2013-01-31 20:53:47.924769774 +0100
@@ -14,7 +14,6 @@
 WARNING_FLAGS = -Wall -Wsign-compare -Wno-invalid-offsetof
 
 repsnapper_CPPFLAGS = \
-	-I$(LIB_DIR)/vmmlib/include \
 	-I$(LIB_DIR)/lmfit/lmfit-3.2/lib \
 	-I$(top_srcdir) \
 	-I$(top_srcdir)/src \
@@ -85,7 +84,7 @@
 
 repsnapper_LDFLAGS = $(EXTRA_LDFLAGS)
 
-repsnapper_LDADD = libreprap.la libclipper.la libpoly2tri.la liblmfit.la libamf.la $(OPENMP_CFLAGS) $(OPENVRML_LIBS) $(GTKMM_LIBS) $(GL_LIBS) $(XMLPP_LIBS) $(LIBZIP_LIBS) $(BOOST_LDFLAGS)
+repsnapper_LDADD = libreprap.la libpoly2tri.la liblmfit.la libamf.la $(OPENMP_CFLAGS) $(OPENVRML_LIBS) $(GTKMM_LIBS) $(GL_LIBS) $(XMLPP_LIBS) $(LIBZIP_LIBS) $(BOOST_LDFLAGS) -lpolyclipping
 
 repsnapperdatadir = $(datadir)/@PACKAGE@
 dist_repsnapperdata_DATA = src/repsnapper.ui
diff -ru a/src/slicer/clipping.h b/src/slicer/clipping.h
--- a/src/slicer/clipping.h	2012-10-16 05:06:27.000000000 +0200
+++ b/src/slicer/clipping.h	2013-01-31 20:47:03.778921196 +0100
@@ -20,7 +20,7 @@
 #pragma once
 #include "types.h"
 
-#include <clipper/clipper/polyclipping/trunk/cpp/clipper.hpp>
+#include <polyclipping/clipper.hpp>
 // see http://angusj.com/delphi/clipper.php
 
 
