diff --git a/libbitdht/src/libbitdht.pro b/libbitdht/src/libbitdht.pro
index dfeb1d1..d70a688 100644
--- a/libbitdht/src/libbitdht.pro
+++ b/libbitdht/src/libbitdht.pro
@@ -1,5 +1,5 @@
 TEMPLATE = lib
-CONFIG += staticlib
+CONFIG += dll
 CONFIG -= qt
 TARGET = bitdht
 QMAKE_CXXFLAGS *= -Wall -DBE_DEBUG
diff --git a/libretroshare/src/libretroshare.pro b/libretroshare/src/libretroshare.pro
index e049ed5..4045d15 100644
--- a/libretroshare/src/libretroshare.pro
+++ b/libretroshare/src/libretroshare.pro
@@ -160,7 +160,7 @@ linux-* {
 
 	DEFINES *= UBUNTU
 	INCLUDEPATH += /usr/include/glib-2.0/ /usr/lib/glib-2.0/include
-	LIBS *= -lgnome-keyring -ldl
+	LIBS *= -lgnome-keyring -ldl -L../../libbitdht/src/lib -lbitdht -L../../openpgpsdk/src/lib -lops
 }
 
 linux-g++ {
diff --git a/openpgpsdk/src/openpgpsdk.pro b/openpgpsdk/src/openpgpsdk.pro
index c7b58ed..e9eb7bc 100644
--- a/openpgpsdk/src/openpgpsdk.pro
+++ b/openpgpsdk/src/openpgpsdk.pro
@@ -1,5 +1,5 @@
 TEMPLATE = lib
-CONFIG = staticlib debug
+CONFIG = dll debug
 
 DEFINES *= OPENSSL_NO_IDEA 
 
diff --git a/retroshare-gui/src/retroshare-gui.pro b/retroshare-gui/src/retroshare-gui.pro
index 636d3ba..093cd26 100644
--- a/retroshare-gui/src/retroshare-gui.pro
+++ b/retroshare-gui/src/retroshare-gui.pro
@@ -56,10 +56,10 @@ linux-* {
 	QMAKE_CXXFLAGS *= -D_FILE_OFFSET_BITS=64
 
 	PRE_TARGETDEPS *= ../../libretroshare/src/lib/libretroshare.a
-	PRE_TARGETDEPS *= ../../openpgpsdk/src/lib/libops.a
+	PRE_TARGETDEPS *= ../../openpgpsdk/src/lib/libops.so
 
+	LIBS += -L../../openpgpsdk/src/lib -lops -lbz2
 	LIBS += ../../libretroshare/src/lib/libretroshare.a
-	LIBS += ../../openpgpsdk/src/lib/libops.a -lbz2
 	LIBS += -lssl -lupnp -lixml -lXss -lgnome-keyring
 	LIBS *= -lcrypto -ldl -lX11 -lz
 
@@ -141,10 +141,10 @@ win32 {
 	#QTPLUGIN += qjpeg
 
 	PRE_TARGETDEPS *= ../../libretroshare/src/lib/libretroshare.a
-	PRE_TARGETDEPS *= ../../openpgpsdk/src/lib/libops.a
+	PRE_TARGETDEPS *= ../../openpgpsdk/src/lib/libops.so
 
 	LIBS += ../../libretroshare/src/lib/libretroshare.a
-	LIBS += ../../openpgpsdk/src/lib/libops.a -lbz2
+	LIBS += -L../../openpgpsdk/src/lib -lops -lbz2
 	LIBS += -L"../../../lib"
 	LIBS += -lssl -lcrypto -lpthreadGC2d -lminiupnpc -lz
 # added after bitdht
@@ -178,7 +178,7 @@ macx {
 
 	CONFIG += version_detail_bash_script
 	LIBS += ../../libretroshare/src/lib/libretroshare.a
-	LIBS += ../../openpgpsdk/src/lib/libops.a -lbz2
+	LIBS += -L../../openpgpsdk/src/lib -lops -lbz2
         LIBS += -lssl -lcrypto -lz 
         #LIBS += -lssl -lcrypto -lz -lgpgme -lgpg-error -lassuan
 	LIBS += ../../../miniupnpc-1.0/libminiupnpc.a
@@ -224,8 +224,8 @@ freebsd-* {
 # ###########################################
 
 bitdht {
-	LIBS += ../../libbitdht/src/lib/libbitdht.a
-	PRE_TARGETDEPS *= ../../libbitdht/src/lib/libbitdht.a
+	LIBS += -L../../libbitdht/src/lib -lbitdht
+	PRE_TARGETDEPS *= ../../libbitdht/src/lib/libbitdht.so
 }
 
 win32 {
diff --git a/retroshare-nogui/src/retroshare-nogui.pro b/retroshare-nogui/src/retroshare-nogui.pro
index 138406d..18d3d7c 100644
--- a/retroshare-nogui/src/retroshare-nogui.pro
+++ b/retroshare-nogui/src/retroshare-nogui.pro
@@ -26,7 +26,7 @@ linux-* {
 	QMAKE_CXXFLAGS *= -D_FILE_OFFSET_BITS=64
 
 	LIBS += ../../libretroshare/src/lib/libretroshare.a
-	LIBS += ../../openpgpsdk/src/lib/libops.a -lbz2
+	LIBS += -L../../openpgpsdk/src/lib -lops -lbz2
 	LIBS += -lssl -lupnp -lixml -lgnome-keyring -lpthread
 	LIBS *= -lcrypto -ldl -lz
 
@@ -72,10 +72,10 @@ win32 {
 	MOC_DIR = temp/moc
 
 	PRE_TARGETDEPS *= ../../libretroshare/src/lib/libretroshare.a
-	PRE_TARGETDEPS *= ../../openpgpsdk/src/lib/libops.a
+	PRE_TARGETDEPS *= ../../openpgpsdk/src/lib/libops.so
 
 	LIBS += ../../libretroshare/src/lib/libretroshare.a
-	LIBS += ../../openpgpsdk/src/lib/libops.a -lbz2
+	LIBS += -L../../openpgpsdk/src/lib -lops -lbz2
 	LIBS += -L"../../../lib" -lssl -lcrypto -lpthreadGC2d -lminiupnpc -lz
 	LIBS += -lssl -lcrypto -lpthreadGC2d -lminiupnpc -lz
 # added after bitdht
@@ -124,7 +124,7 @@ freebsd-* {
 
 # bitdht config
 bitdht {
-	LIBS += ../../libbitdht/src/lib/libbitdht.a
+	LIBS += -L../../libbitdht/src/lib -lbitdht
 }
 
 DEPENDPATH += ../../libretroshare/src
